### PR TITLE
NEXUS-22631: Conan plugin logs warnings on boot

### PR DESCRIPTION
**Pull request by mlukaretkyi**
https://issues.sonatype.org/browse/NEXUS-22631

### CODE REVIEW

File: nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/hosted/ConanHostedRecipe.groovy [hunk 1]
```diff
@@ -86,12 +86,9 @@ class ConanHostedRecipe
   ViewFacet configure(final ConfigurableViewFacet facet) {
     Router.Builder builder = new Router.Builder()
 
-    apiV1.create(builder)
+    addBrowseUnsupportedRoute(builder)
 
-    builder.route(new Route.Builder()
-        .matcher(BrowseUnsupportedHandler.MATCHER)
```
Comment (mcculls) [inline nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/hosted/ConanHostedRecipe.groovy]: Minor: the `BrowseUnsupportedHandler` import at the top of this file can also be removed

File: nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/proxy/v1/ConanProxyRecipe.groovy [hunk 2]
```diff
@@ -91,12 +91,9 @@ class ConanProxyRecipe
   ViewFacet configure(ConfigurableViewFacet facet) {
     Router.Builder builder = new Router.Builder()
 
-    conanApiV1.create(builder);
+    addBrowseUnsupportedRoute(builder)
 
-    builder.route(new Route.Builder()
-        .matcher(BrowseUnsupportedHandler.MATCHER)
```
Comment (mcculls) [inline nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/proxy/v1/ConanProxyRecipe.groovy]: Minor: the `BrowseUnsupportedHandler` import at the top of this file can also be removed

File: nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/hosted/ConanHostedRecipe.groovy (hunk 1 above)
Comment (mlukaretkyi) [reply nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/hosted/ConanHostedRecipe.groovy]: Thanks, fixed https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/92/commits/522a5008e8f1db2847743259d68d93f144979ee6

File: nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/proxy/v1/ConanProxyRecipe.groovy (hunk 2 above)
Comment (mlukaretkyi) [reply nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/proxy/v1/ConanProxyRecipe.groovy]: Thanks, fixed https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/92/commits/522a5008e8f1db2847743259d68d93f144979ee6

**Pull request merged**